types/text-buffer/tslint.json

```diff
@@ -1,11 +1,12 @@
 {
     "extends": "dtslint/dt.json",
     "rules": {
-        // Custom rules.
         "class-name": true,
-        "indent": [true, "tabs"],
+        "indent": [true, "spaces", 4],
         "jsdoc-format": true,
         "max-line-length": [true, 100],
+        "no-single-declare-module": false,
```

Comment:
This rule shouldn't be ignored -- it's better to export things directly from a file than to use `declare module`.